fe/fe-core/src/main/java/org/apache/doris/datasource/HMSExternalCatalog.java

```diff
@@ -69,6 +77,13 @@ public HMSExternalCatalog(long catalogId, String name, String resource, Map<Stri
     @Override
     public void checkProperties() throws DdlException {
         super.checkProperties();
+        // check file.meta.cache.ttl-second parameter
+        String fileMetaCacheTtlSecond = catalogProperty.getOrDefault(FILE_META_CACHE_TTL_SECOND, "");
```

Comment:
User can set `file.meta.cache.ttl-second` = "" and it will pass the check here, although we will not enable ttl with file cache finally, but i think we should let user follow the rules(the value of `file.meta.cache.ttl-second` only can be set as a non-negative number).

Reply:
1. In this case, if user set a negative number , the check will not pass.
2. If user set `file.meta.cache.ttl-second` = "", the check will pass and not enable ttl.

Reply:
Thank you for your advice.
If user can set `file.meta.cache.ttl-second` = "", it is necessary not to pass the check. I will fix it.

Reply:
Yeah, i know it works, but we can distinguish this scene which user set `file.meta.cache.ttl-second` = "" and let user just do not set `file.meta.cache.ttl-second`, well it's depond on you, current implementation is acceptable.

Reply:
Yeah, well, thanks for your advice. I'll think about it.

Reply:
Yeah, well, thanks for your advice. I'll think about it.

Reply:
Thank you for your advice.
If user set `file.meta.cache.ttl-second` = "", the check will not pass check now.